shell2http.go

```diff
@@ -291,7 +294,7 @@ func getShellHandler(appConfig Config, path string, shell string, params []strin
 			shellOut, err = osExecCommand.Output()
 		}
 
-		if err != nil {
+		if err != nil && !appConfig.showErrors {
```

Comment:
Errors are possible to change the html (or image) which outputs command.
Better probably always displays errors in log.